src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
You should create separate scss file for each BEM block(according Checklist point 10)and import them to main.css file